cli/tests/test_new_command.rs

```diff
@@ -400,7 +400,8 @@ fn test_new_insert_before_root() {
     let stderr =
         test_env.jj_cmd_failure(&repo_path, &["new", "--insert-before", "-m", "G", "root()"]);
     insta::assert_snapshot!(stderr, @r###"
-    Error: Cannot insert a commit before the root commit
+    Error: Commit 000000000000 is immutable
+    Hint: Configure the set of immutable commits via `revset-aliases.immutable_heads()`.
```

Comment:
Do you think we can even delete this test case now? It seems to me that it's covered by the tests in `cli/tests/test_immutable_commits.rs`.

Reply:
Well, it's a good illustration for the follow-up commit...

I'm not sure what's best here. If I was writing a new command, I would likely put this test next to the other tests for the command and I might not even notice `test_immutable_commits` exists. I don't mind the duplication very much, but neither does it feel perfectly clean.